### PR TITLE
Moving common code to gvr-unittestutils

### PR DESCRIPTION
**Pull request by parthmehta209**
Moving all common code in gvr-tester and framework-tests into
gvr-unittestutils module.

### CODE REVIEW

File: framework-tests/app/build.gradle [hunk 1]
```diff
@@ -1,12 +1,5 @@
 apply plugin: 'com.android.application'
```
Comment (liaxim) [inline framework-tests/app/build.gradle]: There is a need for a common gradle file but I will take care of it after things settle down.

**Comment by liaxim** [comment]
Looks good. Should we merge?

File: framework-tests/app/build.gradle (hunk 1 above)
Comment (parthmehta209) [inline framework-tests/app/build.gradle]: okay.

**Comment by parthmehta209** [comment]
Yes. We can merge this.

**Comment by NolaDonato** [comment]
I tested this last night with all of my pull requests for the individual test categories and it seems to work OK. I also tested it with a new category and that worked too. We can merge this.

**Pull request merged**